website/src/pages/ProjectCard/ProjectCard.js

```diff
@@ -40,7 +40,7 @@ const ProjectCard = (props) => {
           </div>
         </div>
         <div className="modal_links">
-          <a href="https://github.com/" className="modal_gh_link">
+          <a href={props.link} className="modal_gh_link">
```

Comment:
Can we have `target="_blank" rel="noopener noreferrer"`, so that the project opens in a new tab?

Reply:
Sure sir @sohamsshah I will commit the changes asap :)